source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -66,12 +61,32 @@ int main()
         std::cout << "RotationMatrix:\n" << transformationMatrix.linear() << std::endl;
         std::cout << "TranslationVector:\n" << transformationMatrix.translation() << std::endl;
 
-        // Convert from Zivid to Robot (Transformation Matrix --> any format)
-        const auto robotRotationRepresentations = zividToRobot(transformationMatrix);
-
-        // Convert from Robot to Zivid (any format --> Rotation Matrix)
-        const auto transformationMatrix2 =
-            robotToZivid(robotRotationRepresentations, transformationMatrix.translation());
+        /*
+         * Convert from Zivid to Robot (Transformation Matrix --> any format)
```

Comment:
I don't understand this comment, but that might just be to my poor domain knowledge. What does "any format" mean? Same goes for the similar comment a few lines down.

Reply:
In this case "any format" means "axis-angle, euler/roll pitch yaw, rotation vector and quaternion"

I wanted to keep it a single line. Maybe we can remove it and just keep it as follows:

```
// Extract Rotation Matrix and Translation Vector from Transformation Matrix
std::cout << "RotationMatrix:\n" << transformationMatrix.linear() << std::endl;
std::cout << "TranslationVector:\n" << transformationMatrix.translation() << std::endl;

printHeader("Convert from Zivid (Rotation Matrix) to Robot");
const Eigen::AngleAxisd axisAngle(transformationMatrix.linear());
std::cout << "AxisAngle:\n" << axisAngle.axis() << ", " << axisAngle.angle() << std::endl;
const auto rotationVector = rotationMatrixToRotationVector(transformationMatrix.linear());
std::cout << "Rotation Vector:\n" << rotationVector << std::endl;
const Eigen::Quaterniond quaternion(transformationMatrix.linear());
std::cout << "Quaternion:\n" << quaternion.coeffs() << std::endl;
const auto rpyList = rotationMatrixToRollPitchYawList(transformationMatrix.linear());

printHeader("Convert from Robot to Zivid (Rotation Matrix)");
const auto rotationMatrixFromAxisAngle = axisAngle.toRotationMatrix();
std::cout << "Rotation Matrix from Axis Angle:\n" << rotationMatrixFromAxisAngle << std::endl;
const auto rotationMatrixFromRotationVector = rotationVectorToRotationMatrix(rotationVector);
std::cout << "Rotation Matrix from Rotation Vector:\n" << rotationMatrixFromRotationVector << std::endl;
const auto rotationMatrixFromQuaternion = quaternion.toRotationMatrix();
std::cout << "Rotation Matrix from Quaternion:\n" << rotationMatrixFromQuaternion << std::endl;
rollPitchYawListToRotationMatrix(rpyList);
```

Reply:
Either that, or change it to "Convert from Zivid to Robot coordinates" or something like that?

Reply:
It's not a coordinate representation, but a format for converting between two coordinate systems. 
@SatjaSivcev any opinion?

Reply:
This would be correct:
Convert from Transformation Matrix (Zivid) to position vector and other representations of orientation (Robot)

The rotation matrix is both orientation representation and a tool for converting between two coordinate systems. Same is true for Transformation matrix - it is a representation of a pose (position and orientation) and also a tool for converting between two coordinate systems.